pkg/registry/generic/etcd/etcd.go

```diff
@@ -271,10 +272,21 @@ func (e *Etcd) Update(ctx api.Context, obj runtime.Object) (runtime.Object, bool
 	if err != nil {
 		return nil, false, err
 	}
+	// If the object specified by the user does not have a resource version,
```

Comment:
I don't think that we ever want to encourage this pattern, but it needs to exist, and I'd prefer that it be implemented in the server rather than via a RMW loop in the client. We could create an UpdateOption, but that would be more work, and then we'd need to generate a better error message for the case where resourceVersion == 0 but no forceUpdate option was specified, and we might still need to do something urgently to mitigate false conflicts with status updates.